Code/Editor/AzAssetBrowser/AzAssetBrowserWindow.cpp

```diff
@@ -329,6 +329,15 @@ AzAssetBrowserWindow::AzAssetBrowserWindow(QWidget* parent)
         this,
         [this](const QModelIndex& index)
         {
+            // If multiple AssetBrowsers are open, only the focused browser should perform the rename.
+            QWidget* focusWidget = QApplication::focusWidget();
+
+            auto children = this->findChildren<QWidget*>();
+            if (!children.contains(focusWidget))
+            {
+                return;
+            }
```

Comment:
Could make this even more concise by using https://doc.qt.io/qt-6/qwidget.html#isAncestorOf
```suggestion
            QWidget* focusWidget = QApplication::focusWidget();
            if (!isAncestorOf(focusWidget))
            {
                return;
            }
```